infra/compile.py

```diff
@@ -785,7 +787,12 @@ def compile(tree, ctx, indent=0, patches={}, patchables={}):
         return out
     elif isinstance(tree, ast.ClassDef):
         if tree.decorator_list:
-            assert tree.decorator_list[0].func.value.id == "wbetools"
+            assert isinstance(tree.decorator_list[0], ast.Call) and \
```

Comment:
What is this logic change checking for?

Reply:
Allow a class to be decorated with a hide instruction